data/5e/wizards_of_the_coast/core/players_handbook/ph_classes.lst

```diff
@@ -20,54 +20,54 @@ CLASS:Barbarian	PREMULT:1,[!PRETEMPLATE:1,First Level],[PREABILITY:1,CATEGORY=In
 20	ABILITY:Barbarian Class Feature|AUTOMATIC|Barbarian ~ Primal Champion
 
 
-CLASS:Bard	SOURCEPAGE:p.51	HD:8		TYPE:PC.BASE	MAXLEVEL:20	DEFINE:BardLVL|0	BONUS:VAR|BardLVL|CL	DEFINE:POOL_Bard_Skill|0	DEFINE:BardToolCount|0	BONUS:VAR|POOL_Bard_Skill|1	BONUS:VAR|BardToolCount|1	ABILITY:Internal|AUTOMATIC|TYPE=ArmorProfLight	BONUS:ABILITYPOOL|Skill Proficiency|BardSkillCount	BONUS:ABILITYPOOL|Bard Tool Proficiency|BardToolCount
+CLASS:Bard	SOURCEPAGE:p.51	HD:8		TYPE:PC.BASE	MAXLEVEL:20	DEFINE:BardLVL|0	BONUS:VAR|BardLVL|CL	DEFINE:POOL_Bard_Skill|0	DEFINE:POOL_Bard_Magical_Secrets|0	DEFINE:BardToolCount|0	BONUS:VAR|POOL_Bard_Skill|1	BONUS:VAR|BardToolCount|1	ABILITY:Internal|AUTOMATIC|TYPE=ArmorProfLight	BONUS:ABILITYPOOL|Skill Proficiency|BardSkillCount	BONUS:ABILITYPOOL|Bard Tool Proficiency|BardToolCount
 CLASS:Bard	BONUS:VAR|SpellCasterTotalLVL|BardLVL	BONUS:VAR|SpellCasterClassCount|1
 CLASS:Bard	PREMULT:1,[!PRETEMPLATE:1,First Level],[PREABILITY:1,CATEGORY=Internal,Multiclass Allowed ~ Bard],[PRECLASS:1,Bard=1]
 CLASS:Bard	SPELLTYPE:Arcane	SPELLSTAT:CHA	BONUS:CASTERLEVEL|Bard|CL	MEMORIZE:NO
 ###Block
 1:REPEATLEVEL:1	DONOTADD:SKILLPOINTS
 ###Block
-1	CAST:1,1
-2	CAST:1,1
-3	CAST:1,1
-4	CAST:1,1
-5	CAST:1,1
-6	CAST:1,1
-7	CAST:1,1
-8	CAST:1,1
-9	CAST:1,1
-10	CAST:1,1
-11	CAST:1,1
-12	CAST:1,1
-13	CAST:1,1
-14	CAST:1,1
-15	CAST:1,1
-16	CAST:1,1
-17	CAST:1,1
-18	CAST:1,1
-19	CAST:1,1
-20	CAST:1,1
+1	CAST:0,2
+2	CAST:0,3
+3	CAST:0,4,2
+4	CAST:0,4,3
+5	CAST:0,4,3,2
+6	CAST:0,4,3,3
+7	CAST:0,4,3,3,1
+8	CAST:0,4,3,3,2
+9	CAST:0,4,3,3,3,1
+10	CAST:0,4,3,3,3,2
+11	CAST:0,4,3,3,3,2,1
+12	CAST:0,4,3,3,3,2,1
+13	CAST:0,4,3,3,3,2,1,1
+14	CAST:0,4,3,3,3,2,1,1
+15	CAST:0,4,3,3,3,2,1,1,1
+16	CAST:0,4,3,3,3,2,1,1,1
+17	CAST:0,4,3,3,3,2,1,1,1,1
+18	CAST:0,4,3,3,3,3,1,1,1,1
+19	CAST:0,4,3,3,3,3,2,1,1,1
+20	CAST:0,4,3,3,3,3,2,2,1,1
```

Comment:
You're right now that I think about it, and considering the rules to multiclassing it might be tricky. If you fell this should be ignored that's fine and we can consider how to do this properly later.